ext/libgav1_android.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# This script will build libgav1 for the default ABI targets supported by
+# android. You must pass the path to the android NDK as a parameter to this
+# script.
+#
+# Android NDK: https://developer.android.com/ndk/downloads
+#
+# The git tag below is known to work, and will occasionally be updated. Feel
+# free to use a more recent commit.
+
+if [ $# -ne 1 ]; then
+  echo "Usage: ${0} <path_to_android_ndk>"
+  exit
```

Comment:
Nit: exit 1

because this is an error.

Reply:
Done.